spec/descriptions/tagMaintenanceConfiguration.md

```diff
@@ -0,0 +1,23 @@
+### Maintenance Windows
+Maintenance windows allow you to mute alerts.
+
+### Pausing Maintenance Windows
+Maintenance windows can be paused and resumed.
+
+If a maintenance window is paused, alerts will no longer be muted. 
+
+_NOTE: Pausing unmutes alerts. Maintenance Status is separate from pausing a maintenance window._
+
+### Types of Maintenance Configurations
+- One-Time Maintenance Window
+- Recurrent Maintenance Window
+
+### Maintenance Status
```

Comment:
I would remove this section, as it is not facing an API customer directly, they would rather care about how to put/pause/resume, rather than checking statuses, and this can also be found in the IBM public docs.